.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cppStandard": "c++11",
```

Comment:
Should this be c++14?

Reply:
are we targeting 14?  I though we are using 11

Reply:
c++14 except for two features.

N3652 Extended constexpr
N3653 Default member initializers for aggregates

The notion is to avoid breaking the support of VS2015.